doc/.src/book/dotxt/overview.do.txt

```diff
@@ -279,7 +279,7 @@ solve(a == L, u, bc)
 where `bc` holds information about boundary conditions. This information
 is connected to information about the triangulation, the *mesh*.
 Assuming $u=0$ on the boundary, we can in FEniCS generate a triangular
-mesh over a rectangular domain $[-1,-1]\times [-1,1]$ as follows:
+mesh over a rectangular domain $[-1,1]\times [-1,1]$ as follows:
```

Comment:
Shouldn't this be `[-1,-1] \times [1,1]`?

Reply:
No, they're intervals.